production/promtail-mixin/alerts.libsonnet

```diff
@@ -37,21 +37,6 @@
               |||,
             },
           },
-          {
-            alert: 'PromtailFileLagging',
-            expr: |||
-              promtail_stream_lag_seconds > 10
```

Comment:
We had a user find this metric quite useful recently; can you please remind me why we're removing the metric?

Reply:
OK great :+1: thanks